Ladybird/Headless/Test.cpp

```diff
@@ -490,6 +495,24 @@ ErrorOr<void> run_tests(Core::AnonymousBuffer const& theme, Gfx::IntSize window_
     for (auto const& non_passing_test : non_passing_tests)
         outln("{}: {}", test_result_to_string(non_passing_test.result), non_passing_test.test.input_path);
 
+    if (app.log_slowest_tests) {
```

Comment:
Maybe this could take the N slowest tests, and default to 0?

Reply:
I had thought about that but figured I'd err on the side of the simplicity of just passing the flag without having to pass a value as well. I don't think I'd ever care about e.g. the 15th slowest test, I'd just want to look at the top few slowest

Reply:
Yeah fair enough! Thought it might be interesting to look at but you're right it may not end up being too useful. useful flag though!